src/utils/http-request.ts

```diff
@@ -113,3 +113,11 @@ export function makeHttpRequestData(
     latency ? {latency} : null
   );
 }
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
A comment here may be useful. Do certain common libraries output raw HTTP requests in this format?

Reply:
added descriptive comment